packages/testing/src/execution_testing/cli/pytest_commands/plugins/execute/rpc/remote.py

```diff
@@ -26,15 +26,27 @@ def pytest_addoption(parser: pytest.Parser) -> None:
         dest="rpc_endpoint",
         help="RPC endpoint to an execution client",
     )
-    remote_rpc_group.addoption(
-        "--chain-id",
-        action="store",
-        dest="chain_id",
-        required=False,
-        type=int,
-        default=None,
-        help="ID of the chain where the tests will be executed.",
-    )
+    # Check if --chain-id already exists before adding it
+    # It may already be defined by execute.py plugin
+    chain_id_exists = False
+    for group in parser._groups:
+        for option in group.options:
+            if "--chain-id" in option._long_opts:
+                chain_id_exists = True
+                break
+        if chain_id_exists:
+            break
```

Comment:
I would rather not having to patch like this. The places/plugins where we define `--chain-id` is currently visible in code + the pytest ini files, we could take some time to locate them and simply refactor based on that.